File: src/app/quote.ts
export class Quote{
    id: string;
    symbol: string;
    price: number;
    availableVolume: number;
    expiration: string;
    constructor(id: string, symbol: string, price: number, availableVolume: number, expiration: string){
        this.id = id;
        this.symbol = symbol;
        this.price = price;
        this.availableVolume = availableVolume;
        this.expiration = expiration;
    }
}

File: src/main/java/com/monit/quotemanager/model/Quote.java
package com.monit.quotemanager.model;

import java.util.Date;
import java.util.UUID;

public interface Quote {
    UUID getId();
    void setId();
    void setSymbol(String symbol);
    String getSymbol(String symbol);
    double getPrice();
    void setPrice(double price);
    int getAvailableVolume();
    void setAvailableVolume();
    Date getExpiration();
    void setExpiration(Date expiration);
}
